setup.py

```diff
@@ -37,6 +40,7 @@
         'scripts/custom_linters/tidy-wrapper.sh',
     ],
     install_requires=[
+        'futures',
```

Comment:
This should be installed only for python < 3
As explained in the futures package documentation, you should use the following line instead.
'futures; python_version == "2.7"'

Reply:
Done.